elide-standalone/src/main/java/com/yahoo/elide/standalone/Util.java

```diff
@@ -53,28 +64,61 @@ public static EntityManagerFactory getEntityManagerFactory(String modelPackageNa
             options.put("javax.persistence.jdbc.password", "elide123");
         }
 
-        PersistenceUnitInfo persistenceUnitInfo = new PersistenceUnitInfoImpl("elide-stand-alone",
-                combineModelEntities(modelPackageName, includeAsyncModel), options);
+        dynamicEntityCompiler = new ElideDynamicEntityCompiler(dynamicConfigPath);
+        if (includeDynamicModel) {
+            dynamicEntityCompiler.compile();
+            Collection<ClassLoader> classLoaders = new ArrayList<>();
+            classLoaders.add(dynamicEntityCompiler.getClassLoader());
+            options.put(AvailableSettings.CLASSLOADERS, classLoaders);
+        }
+        PersistenceUnitInfo persistenceUnitInfo = null;
+        try {
+
+            ElideDynamicEntityCompiler.bindClasses = new HashSet<>();
+            // add dynamic generated classes
+            ElideDynamicEntityCompiler.bindClasses.addAll(populateBindClasses(dynamicEntityCompiler, Entity.class));
+            ElideDynamicEntityCompiler.bindClasses.addAll(populateBindClasses(dynamicEntityCompiler, FromTable.class));
+            ElideDynamicEntityCompiler.bindClasses.addAll(populateBindClasses(dynamicEntityCompiler,
+                    FromSubquery.class));
+            // add classes
+            ElideDynamicEntityCompiler.bindClasses.addAll(ClassScanner.getAnnotatedClasses(Entity.class));
```

Comment:
These 3 sets of classes should not be binded to `ElideDynamicEntityCompiler.bindClasses` as they are not compiled through dynamic compiler. These 3 set of classes should be put in `options` as `AvailableSettings.LOADED_CLASSES, <ClassScanner Scanned classes> ` . Refer https://github.com/yahoo/elide/blob/2d6436d07b3818aa897f401a7f8cd9a9b6e97440/elide-spring/elide-spring-boot-autoconfigure/src/main/java/com/yahoo/elide/spring/config/ElideDynamicConfiguration.java#L97